src/pybind/mgr/dashboard/services/custom_banner.py

```diff
@@ -6,22 +6,19 @@
 from .. import mgr
 
 logger = logging.getLogger(__name__)
+ITEM_KEY = 'custom_login_banner'
 
 
-def set_login_banner_mgr(inbuf: str, mgr_id: Optional[str] = None):
-    item_key = 'custom_login_banner'
-    if mgr_id is not None:
-        mgr.set_store(_get_localized_key(mgr_id, item_key), inbuf)
-    else:
-        mgr.set_store(item_key, inbuf)
+def set_login_banner_mgr(inbuf: str):
+    mgr.set_store(ITEM_KEY, inbuf)
```

Comment:
These services look more anemic now. What about removing this file and moving this code inside the CLI commands (encapsulating service logic makes sense when it's complex, but in this case it's almost trivial: the service itself is provided by the `mgr.set|get_store`).